src/vehicle_gui.cpp

```diff
@@ -223,6 +226,118 @@ void BaseVehicleListWindow::BuildVehicleList()
 	this->vscroll->SetCount(static_cast<int>(this->vehgroups.size()));
 }
 
+
+/** Cargo filter functions */
+/**
+ * Check whether a vehicle can carry a specific cargo.
+ * @param v The vehicle to be checked
```

Comment:
Doxygen params don't match the actual function params. 🙂

Reply:
There was no conflict here during the rebase, so this was left out. :P